cloudwatch/main.tf

```diff
@@ -138,14 +138,15 @@ locals {
         "type" : "metric",
         "properties" : {
           "metrics" : [
-            ["AWS/ECS", "MemoryUtilization", "ServiceName", "web", "ClusterName", name, { "label" : name }]
+            ["AWS/ECS", "MemoryUtilization", "ServiceName", "web", "ClusterName", name, { "id" : "m1", "label" : "Max", "stat" : "Maximum" }],
+            ["...", { "id" : "m2", "label" : "Avg", "stat" : "Average" }],
+            ["...", { "id" : "m3", "label" : "p99", "stat" : "p99" }],
           ],
           "view" : "timeSeries",
           "stacked" : true,
```

Comment:
let's set `stacked` to false, stacking these makes no sense:

See e.g. this example, average and max value are both at about 50%, but it seems the average is at 100... 

<img width="595" alt="image" src="https://user-images.githubusercontent.com/20702503/200858511-eb8d9106-a581-4d02-9445-89f3ad55c082.png">